### ISSUE TITLE
Route and Resource Loading: Add `--path` to `beacon.install` generator

### ISSUE THREAD

**Issue by leandrocp** [BeaconCMS/beacon#306]
The `beacon.install` mix task accepts a `--site` option which generates the following scope in the router:

```elixir
  scope "/" do
    pipe_through :browser
    beacon_site "/my_site", site: :my_site
  end
```

We need to add a new optional `--path` option that allows changing the first argument of `beacon_site`, example:

```sh
mix beacon.install --site my_site --path /
```

Would generate:

```elixir
  scope "/" do
    pipe_through :browser
    beacon_site "/", site: :my_site
  end
```

If not specified it takes the value from `--site`.

**Comment by a8t** [comment]
Still looking for help on this?

**Comment by leandrocp** [comment]
> Still looking for help on this?

Yep. A PR is welcome!

**Comment by edborsa** [comment]
https://github.com/BeaconCMS/beacon/pull/387

**Comment by leandrocp** [comment]
Closed by #387 

/cc @AZholtkevych
